libs/parser/src/ast/ast-creator.ts

```diff
@@ -187,6 +193,14 @@ export function astCreatorFactory(BaseCstVisitorClass: CstVisitorBase): CstVisit
       };
     }
 
+    propAccessor(ctx): ExprNodes.PropAccessor {
+      return {
+        type: AstNodeType.PropAccessor,
+        x: null,
```

Comment:
maybe we should add a comment here specifying that the parent should set the value of `x` to itself, I was confused as to why are we setting `x` to `null` and I was the one who wrote the parser initially 😅